app/views/users/new.html.erb

```diff
@@ -0,0 +1,70 @@
+<h1>Create New User</h1>
+
+<!---
+<form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <p>
+    <label for="username">Enter Username</label><br>
+    <input type="text" id="username" name="user[username]">
+  </p>
+  <p>
+    <label for="email">Enter Email</label><br>
+    <input type="email" id="email" name="user[email]">
+  </p>
+  <p>
+    <label for="password">Enter Password</label><br>
+    <input type="password" id="password" name="user[password]">
+  </p>
+  <p>
+    <input type="submit" name="submit" value="Create Account">
+  </p>
+</form>
+-->
+
+<%
+=begin %>
+<%= form_tag("/users", method: "POST") do %>
+  <p>
+  <%= label_tag(:username, "Username") %><br>
+  <%= text_field_tag(:username) %>
+  </p>
+  <p>
+  <%= label_tag(:email, "Email") %><br>
+  <%= email_field_tag(:email) %>
+  </p>
+  <p>
+  <%= label_tag(:password, "Password") %><br>
+  <%= password_field_tag(:password) %>
+  </p>
+  <%= submit_tag("Create Account") %>
+<% end %>
+<%
+=end %>
+
+<%= form_for(@user) do |f| %>
+
+  <% if @user.errors.any? %>
+  <p>The following <%= pluralize(@user.errors.count, 'error')%> prevented your form from being created!</p>
+    <% @user.errors.full_messages.each do |error| %>
+      <p>
+        <%= error %>
+      </p>
+    <% end %>
+  <% end %>
+
+  <p>
```

Comment:
[`Optional`] It would be nice if you put this at the beginning of the file, thus, you can take advantage of it for all the forms.